dev-build/make/make-4.3-r1.ebuild

```
@@ -64,9 +67,14 @@ src_configure() {
 	econf "${myeconfargs[@]}"
 }
 
+src_compile() {
+	emake all
```

Let's just do the latest ebuild please, but for EAPI 8, you can do:
```suggestion
	emake all $(usev doc 'pdf html')
```